hw-js/js/app.js

```diff
@@ -0,0 +1,79 @@
+
+// interpolation- Using a variable to represent a value.
+
+var num = 20
+
+console.log(5 + num);
+
+// concatenation- to combine; put 2 strings together
+
+const pet = "dog";
+const petName = "Eddie";
+
+console.log("I " + " have a " + pet + " and his name is " + petName + "." );
+
+
+// DRY- Don't repeat yourself. We use loops for this, and is an easier way to to repeat.
+
+//BOOLEAN
+
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a < b); 
+console.log(c > d); 
+console.log('Name' === 'Name');
+console.log(a < b && b < c); 
+console.log(a === a && a < d); 
+console.log( e === e);
+console.log( 48 == '48');
+
+
+//section 3- This is an infinite loop, since while is true, it will infinitly console.log
+//section 3-2- This is not an infinite loop, RunProgram var is true, but the console.log is false.
+
+//Reading code- I believe this will make 20 A's, it's reapeating until true.
+
+
+let letters = "A";
+let i = 0;
+
+while (i < 20) {
+	letters += "A";
+	i++;
+}
+
+console.log(letters);
+// made 21 A's, since it's running until A is no longer less than 21
```

Comment:
careful.  again, your answer is correct, but your reasoning is not.  

the loop does not run until a is no longer less than 21, it runs until a is no longer less than 20.
so the loop runs 20 times, from i = 0 through i = 19.  That means it adds 20 As.  Since there was already 1 A in the string, 1 A + 20 As = 21 As.